docs/src/rules/no-extra-parens.md

```diff
@@ -31,6 +31,7 @@ This rule has a string option:
 This rule has an object option for exceptions to the `"all"` option:
 
 * `"conditionalAssign": false` allows extra parentheses around assignments in conditional test expressions
+* `"ternaryOperandBinaryExpressions": false` allows extra parentheses around condition in ternary expressions
```

Comment:
It would be better to insert this after `nestedBinaryExpressions` as they're similar and they fix conflicts with the same rule (`no-mixed-operators`). The order would be:

* conditionalAssign
* returnAssign
* nestedBinaryExpressions
* ternaryOperandBinaryExpressions
* ignoreJSX
* ...

Also, the description isn't accurate. Maybe "allows extra parentheses around binary expressions that are operands of ternary `?:`".